### PR TITLE
added label support to helm charts

### PR DESCRIPTION
**Pull request by omriezra**
added label support to helm charts

### CODE REVIEW

**Comment by niv-ez** [comment]
Add lables to SRA statefulset as well

File: charts/akeyless-csi-provider/values.yaml [hunk 1]
```diff
@@ -128,3 +130,9 @@ csi:
 
   # Pass arbitrary additional arguments to akeyless-csi-provider.
   extraArgs: []
+
+  clusterrole:
```
Comment (liorhasson) [inline charts/akeyless-csi-provider/values.yaml:134]: use camelcase: `clusterRole`

File: charts/akeyless-csi-provider/values.yaml [hunk 2]
```diff
@@ -128,3 +130,9 @@ csi:
...
+    labels: {}
+
+  clusterrolebinding:
```
Comment (liorhasson) [inline charts/akeyless-csi-provider/values.yaml:137]: same

**Review by OriBenHur-akeyless — APPROVED** [review]

**Review by liorhasson — APPROVED** [review]

File: charts/akeyless-csi-provider/values.yaml (hunk 1 above)
Comment (omriezra) [reply charts/akeyless-csi-provider/values.yaml]: fixed

File: charts/akeyless-csi-provider/values.yaml (hunk 2 above)
Comment (omriezra) [reply charts/akeyless-csi-provider/values.yaml]: fixed

**Pull request merged**